customer-commons/src/test/java/no/unit/nva/customer/model/CustomerTest.java

```diff
@@ -47,6 +49,22 @@ public void customerMapperCanMapBetweenCustomerDtoAndCustomerDb() {
         assertNotNull(mappedCustomerDB);
     }
 
+    @Test
+    public void customerMapperCanMapListOfCustomerDtosToCustomerList() {
+        CustomerDto customerDto = createCustomerDb().toCustomerDto();
+        CustomerList customerList = CustomerMapper
+                .toCustomerListFromCustomerDtos(Collections.singletonList(customerDto));
+        assertNotNull(customerList);
+    }
+
+    @Test
+    public void customerMapperCanMapCustomerDbToCustomerDto() {
+        CustomerDb customerDb = createCustomerDb();
+        CustomerDto customerDto = customerDb.toCustomerDto();
+        assertNotNull(customerDto);
+        assertNotNull(customerDto.getContext());
+    }
```

Comment:
this test is covered in CustomerDbTest. Just add a the test that the context is not null